content/documentation/pages/3-stream-developer-guides/2-streams/2-deployment/1-local.md

```diff
@@ -0,0 +1,193 @@
+---
+path: 'stream-developer-guides/streams/deployment/local'
+title: 'Deploy sample stream applications to local host'
+description: 'Deploy sample stream applications to local host '
+---
+
+# Local Deployment
+
+If you have configured and built the [sample stream applications](%currentPath%/stream-developer-guides/streams/standalone-stream-sample) and configured it to run with one of the supported message brokers, you can run them as stand-alone applications in your `local` environment.
+
+<!--TABS-->
```

Comment:
@cppwfs I tried some different ways to organize this. The deployment page for each platform has some common content plus some broker specific content.  I landed on using tabs to avoid duplication, after having six separate docs for deployment.  Thanks for catching the TOC issue. I'm not sure if there is any work around @oodamien . Another problem using tabs is that you can't nest any marker comments,  e.g. nested tabs or even `<!--NOTE-->, <--!TIP-->`, etc. I was willing to make that trade off in this case.